README.md

```diff
@@ -120,8 +120,8 @@ const transaction = await aptos.build.transaction({
 });
 
 // using sign and submit separately
-const senderAuthenticator = aptos.sign.transaction({ signer: alice, transaction });
-const committedTransaction = await aptos.submit.transaction({ transaction, senderAuthenticator });
+const senderAuthenticator = aptos.transaction.sign({ signer: alice, transaction });
+const committedTransaction = await aptos.transaction.submit.simple({ transaction, senderAuthenticator });
```

Comment:
simple, much better than natural.  Though, I now see why had had it the other way.  `simple` is for a single signer?

Reply:
simple is for... a simple transaction. Since everything can be fee payer there is no real only single signer